seedcase_sprout/core/edit_package_properties.py

```diff
@@ -15,16 +15,11 @@ def edit_package_properties(
 ) -> PackageProperties:
     """Edits the properties of an existing package.
 
-    Use this any time you want to edit the package's properties and particularly
-    after using `create_package_structure()`. Because
-    `create_package_structure()` also creates an empty `datapackage.json` file,
-    `edit_package_properties()` is used to fill in the properties file with
-    details specific for the package.
-
-    When you need to edit the `datapackage.json` properties, use this function
-    to ensure the properties are correctly structured and written. It only
-    edits the properties of the package itself, not on the data resources contained
-    within the package.
+    Use this any time you want to edit the package's properties.  When you need
+    to edit the `datapackage.json` properties, use this function to ensure the
+    properties are correctly structured and written. It only edits the
```

Comment:
```suggestion
    properties are correctly structured before it's written. It only edits the
```
I would like it to be more explicit about that this function only edits the properties but doesn't write them. Not sure this suggestion accomplishes that.